applications/nrf5340_audio/doc/user_interface.rst

```diff
@@ -0,0 +1,155 @@
+
+.. _nrf53_audio_app_ui_leds:
+
+LEDs
+****
+
+To indicate the tasks performed, the application uses the LED behavior described in the following table:
+
++--------------------------+-----------------------------------------------------------------------------------------------------------+---------------------------------------------+
+| LED                      |Indication                                                                                                 | Applications                                |
++==========================+===========================================================================================================+=============================================+
+| **LED1**                 | Off - No Bluetooth connection.                                                                            | All                                         |
+|                          +-----------------------------------------------------------------------------------------------------------+---------------------------------------------+
+|                          | Blinking blue on headset: Kits have started streaming audio (BIS and CIS modes).                          | * :ref:`nrf53_audio_broadcast_sink_app`     |
+|                          |                                                                                                           | * :ref:`nrf53_audio_unicast_server_app`     |
+|                          +-----------------------------------------------------------------------------------------------------------+---------------------------------------------+
+|                          | Blinking blue on the CIS gateway: Kit has connected to a headset.                                         | :ref:`nrf53_audio_unicast_client_app`       |
+|                          +-----------------------------------------------------------------------------------------------------------+---------------------------------------------+
+|                          | Blinking blue on the BIS gateway: Kit has started broadcasting audio.                                     | :ref:`nrf53_audio_broadcast_source_app`     |
+|                          +-----------------------------------------------------------------------------------------------------------+---------------------------------------------+
+|                          | Solid blue on the CIS headset: Kits have connected to the gateway.                                        | :ref:`nrf53_audio_unicast_server_app`       |
+|                          +-----------------------------------------------------------------------------------------------------------+---------------------------------------------+
+|                          | Solid blue on the BIS headset: Kits have found a broadcasting stream.                                     | :ref:`nrf53_audio_broadcast_sink_app`       |
```

Comment:
I also think we should do solid first, then blinking, since that is the order they will appear when using the app